packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -123,6 +128,70 @@ class EmbeddedContentConsumer extends React.Component {
     this.addEventListeners(this.props.eventHandlers);
   }
 
+  handleFrameVisualFocusIndicator() {
+    // Selectors for interactable elements
+    const interactableElementSelector = 'a[href]:not([tabindex=\'-1\']), area[href]:not([tabindex=\'-1\']), input:not([disabled]):not([tabindex=\'-1\']), '
+      + "select:not([disabled]):not([tabindex='-1']), textarea:not([disabled]):not([tabindex='-1']), button:not([disabled]):not([tabindex='-1']), "
+      + "[contentEditable=true]:not([tabindex='-1'])";
+
+    // Check if the content is scrollable
+    const isContentScrollable = () => {
+      const doc = this.xfcFrame?.iframe?.contentWindow?.document;
+      return (doc.documentElement.scrollHeight > doc.documentElement.clientHeight
+        || doc.body.scrollHeight > doc.body.clientHeight
+        || doc.documentElement.scrollWidth > doc.documentElement.clientWidth
+        || doc.body.scrollWidth > doc.body.clientWidth);
+    };
+
+    // Check if the iframe has `scrolling` attribute set or not
+    // The default `scrolling` attribute is `auto`.
+    const scrollingEnabled = () => (this.xfcFrame?.iframe?.getAttribute('scrolling') !== 'no');
```

Comment:
Is there a reason that this should not be part of the isContentScrollable method?  Also, do we need this additional check.  If there is nothing to scroll, I would wonder if the setting matters.

Reply:
Yes, we need this additional check because if this is set to 'no', then scrolling isn't available. The `isContentScrollable` changes based on the screen size and available display.

Reply:
So the scrollWidth is greater than the clientWidth even when scrolling is not available?

Reply:
@cm9361 - I looked at `scrollHeight` and `clientHeight` instead since I've vertical scrolling enabled/disabled to check. And yes, even if scrolling is disabled for the iframe, the `scrollHeight` is larger than `clientHeight`.

Reply:
I think the final question then is if this logic should be part of the isContentScrollable method.  It seems like this would all be part of one method.  Is there a scenario where you would not call both?

Reply:
Good question. Yes, we can combine the logic into one function.